.drone.jsonnet

```diff
@@ -297,6 +297,55 @@ local load_artifacts = {
   depends_on: [setup_ci.name],
 };
 
+// builds the extensions
+local extensions_build = {
+  name: 'extensions-build',
+  image: 'ghcr.io/siderolabs/drone-downstream:v1.2.0-33-g2306176',
+  settings: {
+    server: 'https://ci.dev.talos-systems.io/',
+    token: {
+      from_secret: 'drone_token',
```

Comment:
this is currently my personal drone token, better to update with the talos-bot user's drone token